ArcBit/AppDelegate.swift

```diff
@@ -1102,7 +1111,7 @@ import Crashlytics
             }
                 
             else {
-                success(["privateKey":data!])
+                success(["privateKey": data ?? "nil"])
```

Comment:
If data is nil we should do `error("No data")` A nil string can lead to hard to find bugs. See https://stackoverflow.com/questions/4456438/how-to-pass-null-a-real-surname-to-a-soap-web-service-in-actionscript-3